salt/utils/jinja.py

```diff
@@ -749,8 +749,7 @@ class SerializerExtension(Extension, object):
     .. _`import tag`: http://jinja.pocoo.org/docs/templates/#import
     '''
 
-    tags = set(['load_yaml', 'load_json', 'import_yaml', 'import_json',
-                'load_text', 'import_text'])
+    tags = {'load_yaml', 'load_json', 'import_yaml', 'import_json', 'load_text', 'import_text'}
```

Comment:
Looking at the upstream Jinja source code, sets are used for this sort of attribute, so I think the changes made here (and below in this file) are fine.